packages/editor/editor.css

```diff
@@ -39,9 +39,9 @@
 	--layer-overlays-user-brush: 50;
 	--layer-overlays-user-indicator-selected: 60;
 	--layer-overlays-user-indicator-hovered: 70;
-	--layer-overlays-user-handles: 80;
 	--layer-overlays-user-snapline: 90;
 	--layer-overlays-selection-fg: 100;
+	--layer-overlays-user-handles: 105;
```

Comment:
this is probably worth a brief comment: we're gonna forget later why this is a little more subtle